api/client/samples/analysis_kits/stocks_to_use_price_model.py

```diff
@@ -0,0 +1,287 @@
+    ts_out = df_out.fillna(method='bfill', axis=1).iloc[:,0]
+    ts_out.name = 'price_history'
+    
+    return ts_out
+
+def get_price(crop):
+    """calculates monthly average new crop CME price    
+
+    Parameters
+    ----------
+    crop : string
+        'corn' or 'soybeans'
+    
+    Returns
+    -------
+    pandas.Series
+    """
+    ct_mon = ref[crop.lower()]['futures_contract_month']
+    
+    px_hist = contract_month_history(crop, ct_mon)
+    ts_out  = px_hist.resample('M').mean().resample('D').fillna('bfill')
+    ts_out.name = 'price'
+    
+    return ts_out
+
+def get_ending_stocks(crop, region=1215):
+    """Find USDA PS&D's end of marketing year inventory series     
+
+    Parameters
+    ----------
+    crop : string
+        'corn' or 'soybeans'
+    region : int
+        Gro region id
+    Returns
+    -------
+    pandas.Series
+    """
+    
+    c_ref   = ref[crop.lower()]
+    crop_id = c_ref['id']
+
+    client = GroClient(API_HOST, TOKEN)
+    
+    client.add_single_data_series({'metric_id': 1470032, 
+                                   'item_id': crop_id, 
+                                   'region_id': region, 
+                                   'source_id': 14, 
+                                   'frequency_id': 9, 
+                                   'show_revisions': True})
+    
+    df_pts = client.get_df()
+    df_pts = df_pts.sort_values('end_date', ascending=True)
+    df_pts['report_date'] = pd.to_datetime(df_pts['reporting_date'])
+    df_grp = df_pts.groupby('report_date').last()
+    
+    ts_out = df_grp['value']
+    ts_out.name = 'stocks'
+    
+    return ts_out
+
+def get_consumption(crop, region=1215):
+    """Find USDA PS&D's domestic consumption series     
+
+    Parameters
+    ----------
+    crop : string
+        'corn' or 'soybeans'
+    region : int
+        Gro region id
+    Returns
+    -------
+    pandas.Series
+    """
+    c_ref   = ref[crop.lower()]
+    crop_id = c_ref['id']
+    
+    client = GroClient(API_HOST, TOKEN)
+    
+    client.add_single_data_series({'metric_id': 1480032, 
+                                    'item_id': crop_id, 
+                                    'region_id': region, 
+                                    'source_id': 14, 
+                                    'frequency_id': 9,
+                                    'show_revisions': True})
+
+    df_pts = client.get_df()
+    df_pts = df_pts.sort_values('end_date', ascending=True)
+    df_pts['report_date'] = pd.to_datetime(df_pts['reporting_date'])
+    df_grp = df_pts.groupby('report_date').last()
+    
+    ts_out = df_grp['value']
+    ts_out.name = 'use'
+    
+    return ts_out
+
+def get_stocks_to_use(crop):
+    """Calculate historical US carryout to use ratio for a given commodity  
+
+    Parameters
+    ----------
+    crop : string
+        'corn' or 'soybeans'
+    Returns
+    -------
+    pandas.Series
+    """
+    united_states = 1215
```

Comment:
this constant is defined in a few different places, here, in the default arg and in the first function... it would be clearer to just define it once and for all as a global constant

UNITED_STATES_REGION_ID = 1215 

and use that throughout, and remove the default arg value in the get_consumption and get_ending_stocks functions